clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -534,4 +549,21 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
       null
     }
   }
+
+  Map getUserData(BasicGoogleDeployDescription description, String serverGroupName,
+                  String instanceTemplateName, GoogleNamedAccountCredentials credentials) {
+    String customUserData = ''
+    if (description.userDataType && description.userData) {
+      if (description.userDataType.equals("URL")) {
```

Comment:
I'm curious/concerned about the security implications of the URL type user data - 

This probably opens up a not great attack vector if someone can submit a deploy stage that fetches arbitrary content that ultimately ends up in the environment of the launched instance.

Thoughts?